app/test_main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+def test_can_access_google_page(
+        mocked_has_internet_connection: mock.MagicMock,
+        mocked_valid_google_url: mock.MagicMock
+) -> None:
+    can_access_google_page("https://www.google.com.ua")
+    mocked_valid_google_url.assert_called_once()
+    mocked_has_internet_connection.assert_called_once()
```

Comment:
You should check whether can_access_google_page returns the correct value in different cases